consensus/consensus.md

```diff
@@ -25,25 +25,37 @@ of all `0`.
 
 At any point, the `block height` is the distance from the current valid `top` block to the
 genesis block. The genesis block has height zero.
+
 A valid block here means valid according to the consensus rules described below.
+
+We use temporary random leader to build the tree of blocks.
```

Comment:
What exactly is this supposed to mean?

Reply:
My intent was to follow the convention with "tree" and underline that there is a leader that is responsible for part of structure (microblocks). "Temporary" and "random" should underline that this is not a permissioned setup